xbmc/cores/VideoRenderers/LinuxRendererGLES.cpp

```diff
@@ -3046,5 +3050,13 @@ void CLinuxRendererGLES::AddProcessor(CDVDVideoCodecIMXBuffer *buffer, int index
 }
 #endif
 
+bool CLinuxRendererGLES::IsGuiLayer()
+{
+  if (m_format == RENDER_FMT_BYPASS)
```

Comment:
@FernetMenta because that is part of the new rendering and neither upstream nor in a PR yet. it is just my branch that requires "bypass".